src/application/ApplicationTypes.ts

```diff
@@ -18,6 +18,7 @@ export interface Application extends DGraphObject {
  * Interface describing generic Application's Definition model
  */
 export interface AppDefinition extends Definition {
+  uid?: string;
```

Comment:
@dwojno Why not extend `DGraphObject` interface as well here? Is it because you don't want to include the `dgraph.type` property?

Reply:
It looks like `readonly 'dgraph.type' = 'AppDefinition';` is already in `ApplicationDefinitionDTO` class, so should be okay to have `AppDefinition` extend `DGraphObject`

Reply:
Yeah great idea, I'll extend all from `DGraphObject' 👍

Reply:
@jrhender I've updated the PR :)